src/Decoder.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
Hmm, introducing this is problematic, if not a BC break.

I suggest disabling the rule, like we did, for example, in https://github.com/laminas/laminas-validator/blob/23e27edc1520c0bf0c9c7e7f192dd1314677969e/phpcs.xml#L28-L30

Reply:
Done